src/components/PromisesTable.vue

```diff
@@ -79,14 +99,20 @@ export default {
     exclude: { type: Array, default: () => [] }
   },
   data: () => ({
-    search: ''
+    search: '',
+    filterStatus : '',
+    statusOptions : ['All',...promiseStatusValues]
```

Comment:
@weiweitoo  I'm going to remove this 'All' value as when selected no promises would show as it's looking for `promise.status === 'All'`

the element-ui component for selecting promise status is actually clearable